volumes/miovision/sql/table/create-table-discontinuities.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE miovision_api.discontinuities (
+    uid serial PRIMARY KEY,
+    intersection_uid integer,
+    classification_uid integer,
```

Comment:
Should add `REFERENCES` constraints on the appropriate tables.

Reply:
Does that support nulls? 🤷‍♂️

Reply:
It does! (We don't? We should...)

Reply:
Just double checked and `anomalous_ranges` actually does have these constraints already. 
`miovision_qc_intersection_uid_fkey` and
`miovision_qc_classification_uid_fkey`

Reply:
Whoops don't know how I missed those. I will add to this table too.